docs/docs/aleph_zero/sdk.md

```diff
@@ -257,24 +236,101 @@ const sqrtPriceLimit = calculateSqrtPriceAfterSlippage(
 )
```

Comment:
`// get estimated result of swap` -> `// get simulated result of swap` there is no approximation here